src/Compilers/VisualBasic/Portable/Lowering/LocalRewriter/LocalRewriter_Conversion.vb

```diff
@@ -1334,6 +1339,21 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return result
         End Function
 
+        Private Function IsFixInvocation(node As BoundCall) As Boolean
+            If node.Method.Name <> "Fix" Then
```

Comment:
>node.Method.Name <> "Fix" [](start = 15, length = 25)

It is better to explicitly call an API from string so that readers won't be left thinking what Option Compare is in effect and what king of comparison as actually intended here.